manifests/server/config.pp

```diff
@@ -106,6 +108,15 @@
     }
   }
 
+ if ($manage_postgresql_conf == true) {
+    # Prepare the main pg_hba file
+    file { $postgresql_conf_path:
+      owner  => $user,
+      group  => $group,
+      mode   => '0600',
+      }
```

Comment:
@hunner looking for a neat way to properly infer dependance. Ie the directory should be set before we attempt to change the postgresql.conf file.